src/main/java/fashionable/simba/yanawaserver/matching/domain/Participation.java

```diff
@@ -32,18 +31,30 @@ public LocalDateTime getRequestDateTime() {
         return requestDateTime;
     }
 
-    public void changeAcceptedParticipation() {
+    public Participation changeAcceptedParticipation() {
         this.status = ParticipationStatusType.ACCEPTED;
+        return this;
     }
 
-    public Participation(Long id, Long userId, Long matchingId, LocalDateTime requestDateTime, ParticipationStatusType status) {
-        if (matchingId == null) {
-            throw new NoMatchingDataException("매칭 정보를 찾을 수 없습니다.");
+    public Participation changeRejectedParticipation() {
+        this.status = ParticipationStatusType.REJECTED;
+        return this;
+    }
+
+    public Participation(Long id, Long userId, Long recruitmnetId, LocalDateTime requestDateTime, ParticipationStatusType status) {
+        if (recruitmnetId == null) {
+            throw new IllegalArgumentException("모집 정보를 찾을 수 없습니다.");
         }
         this.id = id;
         this.userId = userId;
-        this.matchingId = matchingId;
+        this.recruitmentId = recruitmnetId;
         this.requestDateTime = requestDateTime;
         this.status = status;
     }
+
+    public Participation(Long userId, Long recruitmentId, LocalDateTime requestDateTime) {
+        this.userId = userId;
+        this.recruitmentId = recruitmentId;
+        this.requestDateTime = requestDateTime;
+    }
```

Comment:
```suggestion
    public Participation(Long userId, Long recruitmentId, LocalDateTime requestDateTime) {
        this(null, userId, recruitmentId, requestDateTime, null)
    }
```
위 생성자를 이용해 생성하는 객체들은 유효성 검사를 진행하지 않고 만들어집니다. 특정한 생성자에서 유효한지를 확인하는 로직을 관리하고, 필요에 따라 특정한 생성자를 기반으로 생성자를 만들어야 유효성 검사를 원할하게 진행할 수 있습니다.